components/vf-logo/vf-logo.scss

```diff
@@ -1,31 +1,34 @@
 // vf-logo
 
 .vf-logo {
+  align-items: center;
   color: set-ui-color(vf-ui-color--black);
   display: flex;
   height: 41px;
   text-decoration: none;
 }
 
+.vf-logo__image {
+  --logo-spacing: 8px;
 
-.vf-logo--responsive {
-  background-image: url('../assets/vf-logo/assets/logo-small.svg');
-  background-repeat: no-repeat;
-  background-size: auto 100%;
-  display: flex;
   height: 25px;
-  min-width: 70px;
-}
+  margin-right: 16px; // IE11 Fallback
+  margin-right: var(--logo-spacing);
+
+  @media (min-width: 500px) {
+    --logo-spacing: 12px;
 
-@media (min-width: 500px) {
-  .vf-logo--responsive {
-    background-image: url('../assets/vf-logo/assets/logo-medium.svg');
     height: 28px;
   }
-}
-@media (min-width: 999px) {
-  .vf-logo--responsive {
-    background-image: url('../assets/vf-logo/assets/logo.svg');
+
+  @media (min-width: 999px) {
+    --logo-spacing: 16px;
```

Comment:
should we namespace this to `--vf-logo-spacing` and also should we be explicit? `--vf-logo--margin-right`?

Reply:
Sounds like that fits our methodology, I'll update.

Reply:
As it's a CSS custom property set within the components classname it's localised enough that this shouldn't be a problem.